<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputChannelForm</class>
 <widget class="QWidget" name="inputChannelForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QLabel" name="channelName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="channelGroup"/>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="channelNumber">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="channelMin">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QSpinBox" name="channelMax">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="legend0">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Function</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="legend2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Number</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="legend1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Receiver type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="legend5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="legend3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="legend4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Neutral</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSlider" name="channelNeutral">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
